pkg/controller/controller.go

```diff
@@ -256,3 +261,10 @@ func (c *Controller) kubeNodesInformer(stopCh <-chan struct{}, wg *sync.WaitGrou
 
 	c.nodesInformer.Run(stopCh)
 }
+
+func (c *Controller) mustHaveOperatorServiceAccountInNamespace(namespace string) {
```

Comment:
Just a small complaint - as for me it's too [long](https://projects.haykranen.nl/java/). Maybe give this function some short name, and then explain all the details in commentary? E.g:

```go
// Check that a namespace has a service account in it
func (c *Controller) checkServiceAccount(namespace string) {
}
```

Reply:
your suggestion makes sense, will do